### PR TITLE
docs: separate shell command from output in examples

### PR DESCRIPTION
**Pull request by tylerslaton**
For #48

Quick PR to improve the "copy" experience for our examples. This PR just separates examples commands from their output.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,9 +28,10 @@ the result of that.
 
 When done remove the database file and the downloaded content.
 ```
-```
+```shell
 $ gptscript ./example.gpt
-
+```
```
Comment (g-linville) [inline README.md:33]: I think line 30 needs to be deleted here. This code block starts with two instances of triple backticks.

File: docs/docs/01-overview.md [hunk 2]
```diff
@@ -27,9 +27,10 @@ the result of that.
 
 When done remove the database file and the downloaded content.
 ```
-```
+```shell
 $ gptscript ./example.gpt
-
+```
```
Comment (g-linville) [inline docs/docs/01-overview.md:32]: Same here

**Review by g-linville — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -10,9 +10,9 @@ calls. With GPTScript you can do just about anything, like [plan a vacation](./e
 [edit a file](./examples/add-go-mod-dep.gpt), [run some SQL](./examples/sqlite-download.gpt), or [build a mongodb/flask app](./examples/hacker-news-headlines.gpt).
 
 | :memo: | We are currently exploring options for interacting with local models using GPTScript. |
-|-|:-|
+| ------ | :------------------------------------------------------------------------------------ |
 
-```yaml
+````yaml
```
Comment (g-linville) [inline README.md:15]: This line has an extra backtick, and a couple others do too

Comment (tylerslaton) [reply README.md:15]: Fun fact, this was Zed's auto-formatting 🙃

Comment (g-linville) [reply README.md:15]: Oh lol

**Review by g-linville — APPROVED** [review]

**Review by StrongMonkey — APPROVED** [review]

**Pull request merged**